src/main/webapp/src/app/create-gardens-form/create-gardens.component.html

```diff
@@ -0,0 +1,32 @@
+<mat-card class="mat-elevation-z8 form">
+  <mat-card-title>
+    Create Garden Form
+  </mat-card-title>
+
+  <mat-card-content>
+    <mat-form-field>
+      <input matInput placeholder="Name of Garden" /> </mat-form-field
+    ><br />
+
+    <mat-form-field>
+      <input matInput placeholder="Name of Admin" /> </mat-form-field
+    ><br />
+
+    <mat-form-field>
+      <input matInput placeholder="Address of Garden" /> </mat-form-field
+    ><br />
+
+    <mat-form-field>
+      <textarea matInput placeholder="Garden Bio"></textarea></mat-form-field
+    ><br />
+
+    <mat-form-field>
+      <textarea
+        matInput
+        placeholder="Plants Currently In Garden"
+      ></textarea></mat-form-field
```

Comment:
In future updates (for MVP), should we do a dropdown menu for plants or have a modal window so the user can read more about the plant before adding it to the garden?

Reply:
I like the modal window idea!